tiflash/tiflash-configuration.md

```diff
@@ -253,8 +253,10 @@ delta_index_cache_size = 0
 
 ## Security settings take effect starting from v4.0.5.
 [security]
-    ## New in v5.0. This configuration item enables or disables log redaction. If the configuration value
-    ## is set to true, all user data in the log will be replaced by ?.
+    ## New in v5.0. This configuration item enables or disables log redaction. Value options: true, false, marker. The marker option is introduced in v8.2.0
+    ## The default value is false, which means that log redaction is disabled.
+    ## If the configuration item is set to true, all user data in the log is replaced by `?`.
+    ## If the configuration item is set to marker, all user data in the log is wrapped in `‹ ›`. If the input has `‹`, it is escaped as `‹‹`, and `›` is escaped as `››`. Based on the marked logs, you can decide whether to desensitize the marked information when the logs are displayed.
```

Comment:
```suggestion
    ## If the configuration item is set to marker, all user data in the log is wrapped in `‹ ›`. If user data contains `‹` or `›`, `‹` is escaped as `‹‹`, and `›` is escaped as `››`. Based on the marked logs, you can decide whether to desensitize the marked information when the logs are displayed.
```

Reply:
see https://github.com/pingcap/docs-cn/pull/17627#discussion_r1642424899

Reply:
maybe we should also change the following text "If the input has `‹` ... " here

https://github.com/pingcap/docs/blob/cd719695c040a41c1463f3fdba42f80aed8803d3/system-variables.md?plain=1#L4936